components/Feed.tsx

```diff
@@ -160,12 +179,13 @@ function Feed({
             justifyContent: 'space-between',
           }}
         >
-          {stream?.isConfigured && renderStats()}
```

Comment:
Since we already make some checks inside this functions, I moved this condition there as well.